keyboards/melgeek/mojo75/mojo75.c

```diff
@@ -0,0 +1,158 @@
+		{ 27,     28,        29,       30,       31,       32,       33,       34,       35,         74,        75,         76,       77,        78,        79,        90 }, /* R3 */
+		{ 36,     37,        38,       39,       40,       41,       42,       43,       44,         83,        82,         81,       84,        80,        86,        88 }, /* R4 */
+		{ 45,     46,        47,       48,       49,       NO_LED,   NO_LED,   50,       NO_LED,   NO_LED,      51,         52,       53,        87,        91,        89 }, /* R5 */
+    }, {
+        {0,  0},   {16, 0},   {32, 0},  {48, 0},  {64, 0},  {80, 0},  {96, 0},  {112, 0},  {128, 0},  {144, 0},  {160, 0},  {176, 0},  {192, 0},  {208, 0},   {224, 0},  {240, 0},
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16},  {224, 16}, {240, 16},
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32}, {192, 32}, {208, 32},  {224, 32}, {240, 32},
+		{0, 48},  {16, 48},  {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48}, {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48}, {192, 48}, {208, 48},  {224, 48}, {240, 48},
+		{0, 64},  {16, 64},  {32, 64}, {48, 64}, {64, 64}, {80, 64}, {96, 64}, {112, 64}, {128, 64}, {144, 64}, {160, 64}, {176, 64}, {192, 64}, {208, 64},  {224, 64}, {240, 64},
+		{0, 80},  {16, 80},  {32, 80}, {48, 80}, {64, 80},                     {80, 80},                        {96, 80},  {112, 80}, {128, 80}, {144, 80}, {160, 80},  {176, 80},
+	}, {
+	       4,       4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,          4,         4,
+	       4,       4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,          4,         4,
+	       4,       4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,          4,         4,
+	       4,       4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,          4,         4,
+		   4,       4,         4,        4,        4,        4,        4,        4,         4,         4,         4,          4,        4,        4,          4,         4,
+		   4,       4,         4,        4,        4,                            4,                               4,          4,        4,        4,          4,         4,
+	}
+};
+
+
+void suspend_power_down_kb(void) {
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
+
+void keyboard_pre_init_kb(void) {
+    setPinOutput(A2);
+    writePinHigh(A2);
+    keyboard_pre_init_user();
+}
+#endif
```

Comment:
Given this ties to the config within `keyboards/melgeek/mojo75/rev1/config.h`, this should be moved to `keyboards/melgeek/mojo75/rev1/rev1.c`.

Reply:
> Given this ties to the config within `keyboards/melgeek/mojo75/rev1/config.h`, this should be moved to `keyboards/melgeek/mojo75/rev1/rev1.c`.

Done